src/v2/Apps/Partner/Components/PartnerMeta/index.tsx

```diff
@@ -4,16 +4,19 @@ import { createFragmentContainer, graphql } from "react-relay"
 import { PartnerMeta_partner } from "v2/__generated__/PartnerMeta_partner.graphql"
 import { getENV } from "v2/Utils/getENV"
 import { useRouter } from "v2/System/Router/useRouter"
+import { LocalBusiness } from "v2/Components/Seo/LocalBusiness"
 
 interface PartnerMetaProps {
   partner: PartnerMeta_partner
 }
 
 const PartnerMeta: React.FC<PartnerMetaProps> = ({
   partner: {
-    slug,
+    locationsConnection,
     // @ts-expect-error STRICT_NULL_CHECK
```

Comment:
as a helpful nudge, can we remove these as we come across them? Mostly just adding safe access operators (or guards)

Reply:
Took a shot at this with 620e78d - is that what you had in mind?